src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
1. Well partially, yes. This also has to do with the fact that it's not clear that you'd want height rounded to integer on e.g. react-canvas (if they even care about react-motion).
2. The extra data carried around is _only_ for TransitionMotion. How would you do this:
```jsx
<TransitionMotion styles={stuff}>{interpolated => Object.keys(interpolated).map(style => randomKeyWithMyData...)}
```
Passing as an extra param would work well here, would it?

cc @phaistonian